src/lucky/file_response.cr

```diff
@@ -0,0 +1,81 @@
+class Lucky::FileResponse < Lucky::Response
+  DEFAULT_STATUS = 200
+
+  getter context, path, disposition, filename, debug_message
+
+  def initialize(@context : HTTP::Server::Context,
+                 @path : String,
+                 @content_type : String? = nil,
+                 @disposition : String = "attachment",
+                 @filename : String? = nil,
+                 @status : Int32? = nil,
+                 @debug_message : String? = nil)
+  end
+
+  def print
+    raise Lucky::Exceptions::MissingFile.new("Cannot read file #{path}") unless file_exists?
```

Comment:
Instead of manually passing in the message, what do you think about just passing the path, and setting the full message in the error class similar to the `MissingParam` exceptions?